<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guestAllObjects</class>
 <widget class="QDialog" name="guestAllObjects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>981</width>
    <height>542</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>981</width>
    <height>542</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Наши работы</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twOrg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>531</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Увидеть все объекты</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFind">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите район...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск по району</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список наших работ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
